docs/core/tools/dotnet-publish.md

```diff
@@ -20,8 +20,8 @@ dotnet publish [<PROJECT>|<SOLUTION>] [-a|--arch <ARCHITECTURE>]
     [--manifest <PATH_TO_MANIFEST_FILE>] [--no-build] [--no-dependencies]
     [--no-restore] [--nologo] [-o|--output <OUTPUT_DIRECTORY>]
     [--os <OS>] [-r|--runtime <RUNTIME_IDENTIFIER>]
-    [--self-contained [true|false]] [--no-self-contained]
-     [-s|--source <SOURCE>] [-v|--verbosity <LEVEL>]
+    [--sc|--self-contained [true|false]] [--no-self-contained]
```

Comment:
```suggestion
    [-sc|--self-contained [true|false]] [--no-self-contained]
```

??

Reply:
Two hyphens is correct.  They did it that way because the standard for single-hyphen short names is to use only one character.